app/src/Repository/MsgsRepository.php

```diff
@@ -262,13 +262,13 @@ public function search(?User $user = null, ?int $type = null, ?array $alias = nu
     }
 
     /**
-     * @param ?array{
-     *     sort: string,
-     *     direction: string,
-     * } $sortParams
+     * @param ?array<string, string> $sortParams
      * @return array<int, array<string, mixed>>
      */
-    public function advancedSearch(?User $user = null, string $messageType = 'incoming', ?array $sortParams = null): array
+    public function advancedSearch(
+        ?User $user = null,
+        string $messageType = 'incoming',
+        ?array $sortParams = null): array
     {
```

Comment:
Je précise que je suggère ce genre de changements parce que PHP_CS va nous modifier ça pour être raccord avec la PSR12

```suggestion
        ?array $sortParams = null,
    ): array {
```